sql/core/src/test/scala/org/apache/spark/sql/SparkSessionBuilderSuite.scala

```diff
@@ -42,12 +42,18 @@ class SparkSessionBuilderSuite extends SparkFunSuite {
     assert(initialSession.sparkContext.conf.get("some-config") == "v2")
     assert(initialSession.conf.get("some-config") == "v2")
     SparkSession.clearDefaultSession()
+
+    assert(SparkSession.getDefaultSession === None)
+    assert(SparkSession.getActiveSession === None)
```

Comment:
How about adding a trait `BeforeAndAfterEach` and then extending the function `afterEach` with these logics?